docs/sources/mobile-app/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Grafana OnCall mobile app
+aliases:
+  - /docs/oncall/latest/mobile-app/
+keywords:
+  - Mobile App
+  - oncall
+  - notification
+  - push notification
+weight: 1200
+---
+
+# Grafana OnCall Mobile App
+
+>**Note**: This application is currently in beta
+
+The Grafana OnCall mobile app allows teams to monitor and respond to critical system events from anywhere.
+On-call engineers can reimagine incident response with real-time alerts, on-call notifications, and a growing feature set.
+
+## About Grafana OnCall mobile app
+Grafana OnCall for iOS and Android allows you to:
+
+- Override Do Not Disturb on your device to ensure delivery of critical alerts
+- View and troubleshoot alert details from your mobile device
+- Receive push notifications according to your personal notification settings
+- Login with a quick and secure QR code authorization
+
+The OnCall mobile app allows users to receive push notifications as part of their notification policy.
+Push notifications are one of the many notification options in Grafana OnCall.Regardless of your notification preferences, it’s recommended that you configure a diverse set of notification methods.
+
+The current version of the mobile app includes limited functionality and serves primarily as a notification method. The mobile app is in development, and some features are not yet available.
+Stay tuned, we're working on it!
+
+## Before you begin
+
+The Grafana OnCall mobile app is intended to be used as an extension of your Grafana OnCall instance.
+Grafana OnCall is available for Grafana Cloud and Grafana open source users.
```

Comment:
not enterprise?

Reply:
Not sure about this. The [OnCall product page](https://grafana.com/oss/oncall/) states that the three options are open source, grafana cloud, or cloud pro.